presto-spi/src/main/java/io/prestosql/spi/security/ConnectorIdentity.java

```diff
@@ -47,32 +47,13 @@ public String getUser()
         return role;
     }
 
-    @Override
-    public boolean equals(Object o)
```

Comment:
You could mention in the commit what was refactored. Like:
```
Refactor ConnectorIdentity

- Remove not needed and incorrectly implemented hashCode and equals methods
- Return information about role from toString method.
```

Reply:
`ConnectorIdentity` equality might be used as Map keys. In fact, it _is_ used in our internal code.
Let's talk. @luohao @kokosing

Reply:
I would merge it as it is, if it is related to our internal code we could handle that there.